go.mod

```diff
@@ -67,3 +67,5 @@ require (
 )
 
 go 1.13
+
+replace launchpad.net/gocheck => github.com/go-check/check v0.0.0-20140225173054-eb6ee6f84d0a
```

Comment:
replace？why？

Reply:
`replace` is used to fix https://github.com/apache/dubbo-go/issues/682#issuecomment-670842945.